README.md

```diff
@@ -1,32 +1,18 @@
-# HACBS Release bundles
+# Release Service Catalog
 
-Release service bundles is a curated Tekton bundles catalog that contains bundles that are used by the
-[Appstudio Release operator](https://github.com/redhat-appstudio/release-service) to process releases in the context of
+This repository contains a collection of Tekton resources to be used with the Release Service. The main purpose of these
+resources is to facilitate and streamline the release process.
 AppStudio.
 
-## Using the bundles
+# Resources
+Here's a brief overview of what you can find in the different directories of this catalog:
 
-All the bundles in this repository are pushed to our [Quay organization](https://quay.io/organization/hacbs-release).
-This push happens automatically whenever a new bundle is added to the repository or an existent one is updated. Two new
-tags will be created or updated during this automatic process:
+* `pipelines`: This directory contains Release Pipelines which use a collection of both, Release Tasks and Hub Tasks.
+* `tasks`: The tasks directory holds Tekton Tasks that are used in our Release Pipelines.
+* `hub`: This directory contains Tekton resources that have been sourced from the Tekton Hub.
 
-* **version**: This tag will point to the Catalog version of the Tekton resource, for example "0.1", 0.2", etc..
-* **main**: This tag will point always to the existing code in the repository main branch.
-* **<commit-sha>**: A tag with the commit-sha of the change will be created, so we can have multiple bundles per each
-change in the git repository.
+# Versioning
 
-## Adding new bundles
-
-To add new bundles to this repository, create your resource definition under the appropriate [catalog](catalog)
-subdirectory. Tasks go into [catalog/task](catalog/task), and Pipelines go into [catalog/pipeline](catalog/pipeline).
-The directory name will be used as the bundle name, where the resource type becomes the bundle prefix.
-For example, [catalog/pipeline/hello-world/](catalog/pipeline/hello-world/) would become "pipeline-hello-world".
-Accordingly, [catalog/task/hello-world/](catalog/task/hello-world/) would become "task-hello-world".
-
-> **NOTE:** Although not required, it is recommended to define a single resource per file.
-
-## Linting of yaml files
-
-Whenever a change is pushed to the repository and a pull request is created, a yaml lint task will run to ensure that the
-bundle definition doesn't contain invalid yaml data. Refer to the [.yamllint file](.yamllint) to see the exact applied
-rules. For more information on yamllint, check the [official documentation](https://yamllint.readthedocs.io/en/stable).
+All the resources in this repository are versioned through the use of the `app.kubernetes.io/version` label. Any change
```

Comment:
Can you add more information here? When do we update the changelog? When do we bump the y and when the z in x.y.z? What if it is a breaking change? What if it is not a breaking change? What if we just update the tests or docs?

Reply:
This section contained information about version bumping that is outdated. We need to agree on the promotion strategy and on how to manage changelogs. I was hopping to do that in a separate PR.

Regarding linting, that's an automated process part of the CI. No need to document it. You will get an error and enough information when a PR is created. It was too verbose.

Reply:
I don't see how removing linting documentation is part of removing version directories. If we are waiting to update the docs for a separate PR, we can handle it there. I don't agree with removing it as part of this PR

Reply:
Done.